pennylane_lightning/core/_serialize.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, device_name, use_csingle: bool = False):
         self.hamiltonian_c64 = lightning_ops.observables.HamiltonianC64
         self.hamiltonian_c128 = lightning_ops.observables.HamiltonianC128
 
+        self.use_mpi = False
```

Comment:
Do we want this to be a "private" variable within the class? If so, we can use `self._use_mpi`, assuming we don't need access to this outside the class itself.

Reply:
Nice one!